src/Compilers/Server/VBCSCompilerTests/CompilerServerTests.cs

```diff
@@ -17,39 +18,11 @@ namespace Microsoft.CodeAnalysis.CompilerServer.UnitTests
 {
     public class CompilerServerUnitTests : TestBase
     {
-        private const string CompilerServerExeName = "VBCSCompiler.exe";
-        private const string CSharpClientExeName = "csc.exe";
```

Comment:
This is the huge bit of this change :) - we're no longer starting a process, but rather directly invoking the API in-proc.

Reply:
It looks like the `TestableCompilerServer` is specifically designed for this, while these tests are meant to run as separate binaries. I'm worried that this removes our only testing vector for out-of-proc scenarios.

Reply:
Eh, actually `TestableCompilerServer` looks to be for a limited set of mocking.

But I think it's still valuable to keep the out-of-proc tests. I think we should keep this file as is, mark it desktop-only, and create a new file of in-proc tests (which could be just copy-pasting your changes to a new file.